src/pool/srv_pool.c

```diff
@@ -673,7 +673,6 @@ ds_pool_svc_destroy(const uuid_t pool_uuid)
 	d_rank_list_t	excluded = { 0 };
 	int		rc;
 
-	ds_rebuild_leader_stop(pool_uuid, -1);
```

Comment:
Good, I think the call shouldn't be required here. Why was it added here originally?